cdc/sinkv2/eventsink/mq/mq_sink_test.go

```diff
@@ -54,8 +54,10 @@ func TestWriteEvents(t *testing.T) {
 	t.Parallel()
 
 	kafkav1.NewAdminClientImpl = kafka.NewMockAdminClient
+	newProducerImpl = producer.NewMockProducer
 	defer func() {
 		kafkav1.NewAdminClientImpl = kafka.NewSaramaAdminClient
+		newProducerImpl = producer.NewKafkaProducer
```

Comment:
This pattern might prevent parallel test running if used in multiple test cases.